recipes/metaprokka/meta.yaml

```diff
@@ -31,9 +35,10 @@ test:
 
 about:
   home: https://github.com/telatin/{{ name }}
-  license: GPLv3
+  license: GPLv2
```

Comment:
the SDPX identifier should be `GPL-2.0-only` or `GPL-2.0-or-later`

Reply:
Thanks for spotting

This was a strange diversion I took after seeing prokka's meta.yaml. 
I will double-check, but from what I see prokka itself is GPLv3, if so will PR that.

And ...yes, I know merge is deprecated but sometimes still slips through my fingers :) I wanted to update not merge 😇

Reply:
ah okay i see your latest commit. what i meant was that the abbreviation should be `GPL-3.0-only` or `GPL-3.0-or-later` to disambiguate the two possibilities instead of `GPLv3`